helm-charts/stable/amazon-eks-pod-identity-webhook/templates/job-CronJob.yaml

```diff
@@ -0,0 +1,41 @@
+# Copyright 2020 Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  namespace: default
+  name: pod-identity-webhook-job
+spec:
+  schedule: "@weekly"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          serviceAccountName: pod-identity-webhook-job
+          containers:
+          - name: job
+            image: public.ecr.aws/amazonlinux/amazonlinux:2
```

Comment:
This seems like a sound default, but can you make this overrideable in the `values.yaml` file?